modules/aruco/include/opencv2/aruco/aruco_calib_pose.hpp

```diff
@@ -0,0 +1,287 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+#ifndef __OPENCV_ARUCO_CALIB_POSE_HPP__
+#define __OPENCV_ARUCO_CALIB_POSE_HPP__
+#include <opencv2/aruco/board.hpp>
+#include <opencv2/calib3d.hpp>
+
+namespace cv {
+namespace aruco {
+
+//! @addtogroup aruco
+//! @{
+
+/** @brief rvec/tvec define the right handed coordinate system of the marker.
+ * PatternPos defines center this system and axes direction.
+ * Axis X (red color) - first coordinate, axis Y (green color) - second coordinate,
+ * axis Z (blue color) - third coordinate.
+ * @sa estimatePoseSingleMarkers(), @ref tutorial_aruco_detection
+ */
+enum PatternPos {
+    /** @brief The marker coordinate system is centered on the middle of the marker.
+     * The coordinates of the four corners (CCW order) of the marker in its own coordinate system are:
+     * (-markerLength/2, markerLength/2, 0), (markerLength/2, markerLength/2, 0),
+     * (markerLength/2, -markerLength/2, 0), (-markerLength/2, -markerLength/2, 0).
+     *
+     * These pattern points define this coordinate system:
+     * ![Image with axes drawn](images/singlemarkersaxes.jpg)
+     */
+    CCW_CENTER,
```

Comment:
Non-class enum values should be properly prefixed to avoid conflicts.

Reply:
add `ARUCO_` prefix